src/pageLayout/containers/TreeNav.tsx

```diff
@@ -31,179 +32,197 @@ import './TreeNav.scss'
+            </TreeNav.Item>
+          )
+        })}
+        {CLOUD && isFlagEnabled('helpBar') && (
+          <TreeNav.Item
+            id="support"
+            testID="nav-item-support"
+            icon={<Icon glyph={IconFont.QuestionMark_New} />}
+            label="Help &amp; Support"
+            shortLabel="Support"
+            className="helpBarStyle"
+          >
+            <TreeNav.SubMenu position={PopoverPosition.ToTheRight}>
+              <TreeNav.SubHeading label="Support" />
+              <TreeNav.SubItem
+                id="documentation"
+                label="Documentation"
+                testID="nav-subitem-documentation"
+                linkElement={() => (
+                  <SafeBlankLink
+                    href="https://docs.influxdata.com/"
+                    onClick={() => handleEventing('documentation')}
+                  />
+                )}
+              />
+              <TreeNav.SubItem
+                id="faqs"
+                label="FAQs"
+                testID="nav-subitem-faqs"
+                linkElement={() => (
+                  <SafeBlankLink
+                    href="https://docs.influxdata.com/influxdb/cloud/reference/faq/"
+                    onClick={() => handleEventing('faq')}
+                  />
+                )}
+              />
+
+              {isFlagEnabled('helpBarSfdcIntegration') && (
                 <TreeNav.SubItem
-                  id="influxdbSlack"
-                  label="InfluxDB Slack"
-                  testID="nav-subitem-influxdb-slack"
-                  linkElement={() => (
-                    <SafeBlankLink href="https://influxcommunity.slack.com/join/shared_invite/zt-156zm7ult-LcIW2T4TwLYeS8rZbCP1mw#/shared-invite/email" />
-                  )}
+                  id="contactSupport"
+                  label="Contact Support"
+                  testID="nav-subitem-contact-support"
+                  onClick={handleSelect}
                 />
-              </TreeNav.SubMenu>
-            </TreeNav.Item>
-          ) : null}
-        </TreeNav>
-      </OrgSettings>
-    )
-  }
+              )}
+              <TreeNav.SubHeading label="Community" />
+              <TreeNav.SubItem
+                id="offcialForum"
+                label="Official Forum"
+                testID="nav-subitem-forum"
+                linkElement={() => (
+                  <SafeBlankLink
+                    href="https://community.influxdata.com"
+                    onClick={() => handleEventing('officialForum')}
+                  />
+                )}
+              />
+              <TreeNav.SubItem
+                id="influxdbSlack"
+                label="InfluxDB Slack"
+                testID="nav-subitem-influxdb-slack"
+                linkElement={() => (
+                  <SafeBlankLink href="https://influxcommunity.slack.com/join/shared_invite/zt-156zm7ult-LcIW2T4TwLYeS8rZbCP1mw#/shared-invite/email" />
+                )}
+              />
```

Comment:
there should be a Feedback and Questions SubItem right under the slack link. Here is the code 
```
<TreeNav.SubHeading label="Feedback" />
                <TreeNav.SubItem
                  id="feedback"
                  label="Feedback & Questions"
                  testID="nav-subitem-feedback-questions"
                  onClick={openFeedbackOverlay}
                />
```